nextcloudappstore/api/v1/release/info.xsd

```diff
@@ -242,6 +242,13 @@
         </xs:restriction>
     </xs:simpleType>
 
+    <xs:simpleType name="doc-url">
+        <xs:restriction base="xs:anyURI">
+            <xs:pattern value="(https://.+|(admin|user|developer)-[a-z]+[\-a-z]*[a-z]+)"/>
```

Comment:
Supported with 7+ 🙈

Reply:
base="xs:anyURI" is now wrong, should be base="non-empty-string"

Reply:
**admin-** prefix should probably only work in \<admin> tags right?

Reply:
Well it's not enforced on server side, but yeah it's the case at the moment, so I guess we can do that.